core/site.go

```diff
@@ -182,7 +182,14 @@ func NewSiteFromConfig(
 	}
 
 	if len(site.batteryMeters) > 0 && site.ResidualPower <= 0 {
-		site.log.WARN.Println("battery configured but residualPower is missing (add residualPower: 100 to site)")
+		if site.ResidualPower == 0 {
+			site.log.WARN.Println("ResidualPower is set to 100, because battery is configured and residualPower is missing or = 0.")
```

Comment:
Diese Logmeldung können wir uns mMn. dann sparen. Der normale Nutzer sollte gar nichts über `residualPower` wissen müssen und mit den Default gut fahren. Wenn er da doch was dran tweaken möchte findet er den Parameter und seiner Bedeutung über die Doku.

Einziger Punkte wäre, wenn er ihn explizit auf 0 setzt um ein bestimmtes Verhalten zu bekommen und wir seinen Wert dann überschreiben. Wir können hier ja leider nicht zwischen `0` und `nicht gesetzt` unterscheiden. Halte das aber für einen Edge-Case der auch eher in die Doku sollte.

Reply:
Hmmm ... da bin ich jetzt hin- und hergerissen. 
Einerseits gebe ich dir recht
>der normale Nutzer sollte gar nichts über residualPower wissen müssen 

andererseits finde ich es gut, wenn über "Systemeingriffe" informiert wird.
Gibt es auch `log.Info`?

Reply:
> Wir können hier ja leider nicht zwischen 0 und nicht gesetzt unterscheiden.

@naltatis die Doku von Viper behauptet wir können das. 

> Make it easy to tell the difference between when a user has provided a command line or config file which is the same as the default.

Liefert isSet() ggf. false wenn ein Default gezogen wird?